tests/ssl-opt.sh

```diff
@@ -1628,6 +1628,16 @@ run_test    "Context serialization, re-init, both serialize, with CID" \
             -c "Deserializing connection..." \
             -s "Deserializing connection..."
 
+requires_config_enabled MBEDTLS_SSL_CONTEXT_SERIALIZATION
+run_test    "Saving the serialized context to a file" \
+            "$P_SRV dtls=1 serialize=1 context_file=context_srv.txt" \
+            "$P_CLI dtls=1 serialize=1 context_file=context_cli.txt" \
+            0 \
+            -s "Save serialized context to a file... ok" \
+            -c "Save serialized context to a file... ok"
+rm -f context_srv.txt
+rm -f context_cli.txt
```

Comment:
Could we move this to the `cleanup()` function instead?

Reply:
This should be copied to cleanup but not moved. It seems that cleanup is not triggered when the test completes normally but only when certain signals from the system occur:
`trap cleanup INT TERM HUP`

Reply:
I added it to cleanup